index.js

```diff
@@ -1,66 +1,190 @@
+                    '_pageFailed',
+                    '_pageCrashed',
+                ],
+                allowAfterThis: true,
+            }],
+            // ++ and -- are fine.
+            'no-plusplus': 'off',
+            quotes: ['error', 'single', {
+                avoidEscape: true,
+                allowTemplateLiterals: true,
+            }],
+            // Disable as forcing names for function expression doesn't bring much value.
+            'func-names': 'off',
+            // Disable as this rule is overly strict and can cause breaking changes.
+            'class-methods-use-this': 'off',
+            // Forcing "use strict" is not necessary.
+            strict: 'off',
+            // We don't want to force newlines before and after curly braces.
+            'object-curly-newline': 'off',
+            // It's fine to use await in loops, e.g. for loop with awaits in it for sequential execution.
+            'no-await-in-loop': 'off',
+            // It's fine to use continue in loops.
+            'no-continue': 'off',
+            'arrow-body-style': 'off',
+            'no-restricted-syntax': ['error', 'ForInStatement', 'LabeledStatement', 'WithStatement'],
+            'no-prototype-builtins': 'off',
+            // Set max line length to 160 characters.
+            'max-len': ['error', {
+                code: 160,
+                ignoreUrls: true,
+                ignoreTemplateLiterals: true,
+            }],
+            // Allow less strict formatting of function parentheses.
+            'function-paren-newline': 'off',
+            // Prefer destructuring for certain cases only. Forcing destructuring for arrays can make the code less readable.
+            'prefer-destructuring': ['error', {
+                VariableDeclarator: {
+                    array: false,
+                    object: true,
+                },
+                AssignmentExpression: {
+                    array: false,
+                    object: false,
+                },
+            }, {
+                enforceForRenamedProperties: false,
+            }],
+            // So that the functions calling return show up in stack traces in case of an error.
+            'no-return-await': 'off',
+            // Do not enforce newlines between class members if they have a single line.
+            'lines-between-class-members': ['error', 'always', {
+                exceptAfterSingleLine: true,
+            }],
+
+            // Rules related to eslint-plugin-import.
+            // Force external modules to be specified in the package.json.
+            'import/no-extraneous-dependencies': ['error', {
+                devDependencies: true,
+            }],
+            // Force the use of named exports.
+            'import/no-default-export': 'error',
+            // It's ok to not have a default export and we force named exports anyway.
+            'import/prefer-default-export': 'off',
+            'import/no-unresolved': 'off',
```

Comment:
Why disable this one?

Reply:
It was in the original config, see https://github.com/apify/apify-eslint-config/blob/master/index.js#L29
I'd be happy to remove it. Let's see what others think.

Reply:
Removed 😎